src/erlcloud_states.erl

```diff
@@ -573,7 +573,7 @@ stop_execution(ExecutionArn, Options) ->
     stop_execution(ExecutionArn, Options, default_config()).
 
 -spec stop_execution(ExecutionArn   :: binary(),
-                     Options           :: list(),
+                     Options           :: list() | map(),
```

Comment:
Same here, `list()` looks like just a bad type that should be removed

Reply:
Yes

Reply:
Fixed in 067807c.